.github/workflows/release-artifacts.yml

```diff
@@ -51,19 +63,29 @@ jobs:
         run: echo "version=$(./gradlew -q showVersion | tr -d '[:space:]')" >> "${GITHUB_OUTPUT}"
 
       - name: Validate Release
+        if: ${{ github.event_name == 'push' }}
         run: |
           if [[ "${{ steps.tag.outputs.version }}" != "${{ steps.project.outputs.version }}" ]]; then
             echo "::error file=version.gradle,line=5,title=Version Mismatch::Tag version '${{ steps.tag.outputs.version }}' does not match the Gradle project version '${{ steps.project.outputs.version }}'. Please update the 'version.gradle' file before tagging."
             exit 1
           fi
 
+      - name: Validate Release (workflow_dispatch)
+        if: ${{ github.event_name == 'workflow_dispatch' }}
+        run: |
+          if [[ "${{ steps.workflow_tag.outputs.version }}" != "${{github.event.inputs.version}}" ]]; then
```

Comment:
Pointless comparison, should remove this step for workflow dispatch triggered jobs.

Reply:
Removed this step.

Reply:
Removed this step.